packages/core/src/checkout-buttons/checkout-button-strategy-action-creator.ts

```diff
@@ -57,7 +57,15 @@ export default class CheckoutButtonStrategyActionCreator {
                 ),
                 this._paymentMethodActionCreator.loadPaymentMethod(
                     options.methodId,
-                    { timeout: options.timeout, useCache: true }
+                    {
+                        ...options.currencyCode && {
```

Comment:
Can we move this outside of here, just to make it easier to read.

Reply:
Sure